src/register_target.cpp

```diff
@@ -11,7 +12,16 @@ std::unordered_map<std::string, target>& target_map()
 }
 
 void register_target(const target& t) { target_map()[t.name()] = t; }
-target make_target(const std::string& name) { return target_map().at(name); }
+
+target make_target(const std::string& name)
+{
+    if(target_map().find(name) == target_map().end())
+    {
+        MIGRAPHX_THROW("Requested target " + name + " is not enabled or not supported");
```

Comment:
It would be good to put single quotes around the target name.

Reply:
Done